internal/driver/nodeserver_helpers.go

```diff
@@ -328,39 +328,24 @@ func (ns *NodeServer) mountVolume(ctx context.Context, devicePath string, req *c
 // If not, it formats the device using the provided LuksContext.
 // Finally, it prepares the LUKS volume for mounting.
 func (ns *NodeServer) prepareLUKSVolume(ctx context.Context, devicePath string, luksContext LuksContext) (string, error) {
```

Comment:
nit: Could we change the name of this func to something more focused? example: formatLuksVolume()

Reply:
implemented